_posts/2016-08-27-Milestone-on-OSS-by-Default.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Helping the Web towards OSS by Default"
+date: 2016-08-27 12:17
+author: orta
+categories: [force, node, microgravity, js, JavaScript web]
+series: Open Source by Default
+---
+
+The main Artsy.net website for the desktop, [Force][force_gh], was our first Artsy application to open it's source code, [Craig][craig] and [Brennan][brennan] did it [back in 2014][force_oss]. Force's public offering laid the groundwork for the iOS OSS projects to come afterwards: [Eidolon][eidolon_oss], [Eigen][eigen_oss], [Energy][energy_oss] and [Emergence][emergence_oss].
+
+Force wasn't quite Open Source by Default though, it represented a _really_ important step for  Artsy's OSS perspective but was not the end goal. We were opening our source, but not opening our process.
+
+This month both [Force][force_gh], the desktop version of Artsy.net and [Microgravity][mg_gh], the mobile version - moved to being built entirely in the open. Read on to find out how.
+
+<!-- more -->
+
+## Force
+
+Over the course of the last month, I've sat on and off with Charles "[Cab][cab]" Brokskoski, and figured out what it would take to migrate Force to work in the public. Previous to this, work happened on an a private repo, and we would push that code to the public.
+
+We scoped out what it would require, creating an issue that summerizing the work. Then we waited for 2 weeks, to give people the chance to discuss the idea and to offer examples for why we should delay or not move. 
```

Comment:
I like this point a lot. Might be worth reinforcing why we did it and why it's important.